build-plugins/build-support/src/main/kotlin/aws/sdk/kotlin/gradle/plugins/artifactsizemetrics/AnalyzeArtifactSizeMetrics.kt

```diff
@@ -109,34 +110,73 @@ internal abstract class AnalyzeArtifactSizeMetrics : DefaultTask() {
         val significantChange: Boolean,
         val hasDelta: Boolean,
     )
 
-    private fun createDiffTable(analysis: ArtifactSizeMetricsAnalysis): String = buildString {
+    private fun createDiffTables(analysis: ArtifactSizeMetricsAnalysis): String = buildString {
         appendLine("Affected Artifacts\n=")
-        appendLine("| Artifact |Pull Request (bytes) | Latest Release (bytes) | Delta (bytes) | Delta (percentage) |")
-        appendLine("| -------- | ------------------: | ---------------------: | ------------: | -----------------: |")
-        analysis.metrics.forEach { metric ->
-            if (metric.value.delta != 0L) {
-                append("|")
-                append(metric.key)
-                append("|")
-                if (metric.value.currentSize == 0L) append("(does not exist)") else append("%,d".format(metric.value.currentSize))
-                append("|")
-                if (metric.value.latestReleaseSize == 0L) append("(does not exist)") else append("%,d".format(metric.value.latestReleaseSize))
-                append("|")
-                append("%,d".format(metric.value.delta))
-                append("|")
-                append("%.2f".format(metric.value.percentage))
-                append("%")
-                appendLine("|")
+
+        val requiresAttention = StringBuilder()
+            .appendLine("Significantly increased in size")
+            .appendLine()
+            .append(tableHeader)
+        var artifactRequiresAttention = false
+
+        val ordinary = StringBuilder()
+            .appendLine("<details>") // See: https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/organizing-information-with-collapsed-sections
+            .appendLine("<summary>Changed in size</summary>")
+            .appendLine()
+            .append(tableHeader)
+        var artifactOrdinaryChange = false
+
+        analysis.metrics
+            .toList()
+            .sortedBy { it.second.percentage }.toMap()
+            .forEach { metric ->
+                if (metric.value.delta.isNotaFluctuation()) {
+                    val row = buildString {
+                        append("|")
+                        append(metric.key)
+                        append("|")
+                        if (metric.value.currentSize == 0L) append("(does not exist)") else append("%,d".format(metric.value.currentSize))
+                        append("|")
+                        if (metric.value.latestReleaseSize == 0L) append("(does not exist)") else append("%,d".format(metric.value.latestReleaseSize))
+                        append("|")
+                        append("%,d".format(metric.value.delta))
+                        append("|")
+                        append("%.2f".format(metric.value.percentage))
+                        append("%")
+                        if (metric.value.requiresAttention()) append("⚠️")
+                        appendLine("|")
```

Comment:
**Question**: Is the ⚠️ necessary now that there are two tables? Won't _every_ entry in the **Significantly increased in size** table have the ⚠️ next to it?

Reply:
Yeah, I think it looks good but I also see how it can be an eye sore. I'm open to getting rid of it

Reply:
I think it's just redundant information, I support removing it